tests/test_imports.py

```diff
@@ -0,0 +1,23 @@
+"""Module that tests the imports of the package."""
+
+
+def test_import():
+    import lablink_client
+
+    from lablink_client import crd_connect
+    from lablink_client import database
+    from lablink_client import logging_utils
+    from lablink_client import subscribe_instance
+    from lablink_client import update_inuse_status
+
+
+if __name__ == "__main__":
+
+    # Add the parent directory to the pythonpath (pytest setup to do this automatically)
+    import os
+    import sys
+
+    sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+    # Run the test
+    test_import()
```

Comment:
With the changes in [b8005](https://github.com/talmolab/cosyne_vm/pull/5/commits/b8005b74e0eaa3488a4ae2ad0f15e43aecb174d1), I believe we can remove this `pythonpath` manipulation. Also we don't really need debug code in here.
```suggestion
```